<launch>
<!--
	<node pkg="guidance" type="guidanceNode" name="guidanceNode" output="screen" />

	<include file="$(find dji_sdk)/launch/sdk_manifold.launch"/>
-->
	<node pkg="uav" type="uav" name="uav" output="screen">
	<param name="spin_rate"           type="int"    value="50"/>
	<param name="serial_port"         type="string" value="/dev/ttyUSB0"/>  
  	<param name="serial_baudrate"     type="int"    value="115200"/>
	<param name="takeoff_height"      type="double"  value="1.0"/>
	<param name="landing_height"      type="double"  value="0.3"/>
	<param name="xy_err_tolerence"    type="double"  value="0.1"/>
	<param name="z_err_tolerence"     type="double"  value="0.1"/>
	<param name="yaw_err_tolerence"   type="double"  value="0.1"/>
	<param name="vision_pos_coeff"    type="double"  value="0.001"/>
	<rosparam command="load" file="$(find uav)/config/pid.yaml" />
	<rosparam command="load" file="$(find uav)/config/dropoint.yaml" />
	<remap from="/guidance/odom" to="/dji_sdk/odometry" />
	</node>

</launch>

